src/legacy/core_plugins/data/public/query/query_bar/components/query_bar_input.tsx

```diff
@@ -498,28 +509,23 @@ export class QueryBarInputUI extends Component<Props, State> {
                 }}
                 autoComplete="off"
                 spellCheck={false}
-                aria-label={
-                  this.props.screenTitle
-                    ? this.props.intl.formatMessage(
-                        {
-                          id: 'data.query.queryBar.searchInputAriaLabel',
-                          defaultMessage:
-                            'You are on search box of {previouslyTranslatedPageTitle} page. Start typing to search and filter the {pageType}',
-                        },
-                        {
-                          previouslyTranslatedPageTitle: this.props.screenTitle,
-                          pageType: this.props.appName,
-                        }
-                      )
-                    : undefined
-                }
+                aria-label={this.props.intl.formatMessage(
+                  {
+                    id: 'data.query.queryBar.searchInputAriaLabel',
+                    defaultMessage: 'Start typing to search and filter the {pageType} page',
```

Comment:
If you change the default message of a string, you should remove all existing translations for this string, so it will be retranslated by our translators again. Just search for that id and remove it from all internationalization files.

Reply:
Looks like all of them were already removed... Am I missing anything?